src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -199,6 +199,23 @@ trait ResourceRoutes extends SamUserDirectives with SecurityDirectives with SamM
                     }
                   }
                 } ~
+                pathPrefix("memberPolicies") {
+                  requireActionsForSharePolicy(policyId, samUser, samRequestContext) {
```

Comment:
I know there is some debate over this. I feel pretty strongly that this should assert the calling user has at least READ permission on the memberResource to-be-added.